__tests__/unit/chartConfigService.test.js

```diff
@@ -5,7 +5,7 @@ describe('c-chart-config-service', () => {
   test(`Default configuration`, () => {
```

Comment:
**suggestion:** maybe a test to ensure call to defaultConfiguration end with different object

The test could compare two defaultConfiguration call this way:
- === does not work
- == work

Reply:
Yes good idea !

Reply:
What I meant is that we should test that calling `ChartConfigService.defaultConfiguration()` does not return a shallow copy but a new Object instance (with the same value, right, but a new instance of the same data)